numpyro/util.py

```diff
@@ -72,6 +72,20 @@ def fori_loop(lower, upper, body_fun, init_val):
         return lax.fori_loop(lower, upper, body_fun, init_val)
 
 
+def lax_map(f, xs):
```

Comment:
Since this is a very particular use case, let us keep it private to mcmc class, or add some docstring so that this isn't used accidentally instead of `lax.map`.

Reply:
yeah, that makes sense